src/components/EvSections/SponsorsImageGridSection.jsx

```diff
@@ -0,0 +1,70 @@
+import { Box } from "@mui/system";
+import React from "react";
+import { Grid } from "@mui/material";
+import SponsorImage from "components/EvComponents/SponsorImage";
+import { SectionTitle, BlockTitle } from "components/EvComponents/StyledTypography";
+
+const sponsorsImagesData = [
+    {
+        source: "/assets/images/brands/tesla.png",
+    },
+    {
+        source: "/assets/images/brands/apple.png",
+    },
+    {
+        source: "/assets/images/brands/bmw.png",
+    },
+    {
+        source: "/assets/images/brands/dell.png",
+    },
+    {
+        source: "/assets/images/brands/ferrari.png",
+    },
+    {
+        source: "/assets/images/brands/ray-ban.png",
+    },
+    {
+        source: "/assets/images/brands/lotto.png",
+    },
+    {
+        source: "/assets/images/brands/toyota.png",
+    },
+    {
+        source: "/assets/images/brands/xiaomi.png",
+    },
+    {
+        source: "/assets/images/brands/samsung.png",
+    },
+    {
+        source: "/assets/images/brands/sony.png",
+    },
+    {
+        source: "/assets/images/brands/occular.png",
+    },
+    {
+        source: "/assets/images/brands/levis.png",
+    },
+    {
+        source: "/assets/images/brands/amazon.png",
+    },
+]
+
+const SponsorsImageGrid = () => {
+    return (
+        <Box sx={{ textAlign: "center", }}>
+            <SectionTitle>Event Sponsors</SectionTitle>
+            <Grid container columns={{ xs: 1, sm: 3, md: 6 }} sx={{ padding: "35px", textAlign: "center", placeItems: "center", }}>
```

Comment:
this is not the right way to use the grid.... leave the container grid with 12 columns all the time
and set the item grids to {xs:12 , sm:4 , md:2}
contact me if you don't understand this point

Reply:
![image](https://user-images.githubusercontent.com/28100885/181864120-2368acd5-abce-4a6e-bb83-c504984a8f13.png)

Reply:
how can I solve this overlap?

Reply:
solved